code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -431,6 +431,19 @@
 		/obj/item/weapon/melee/sabre
 		)
 
+
+
+/obj/item/weapon/storage/belt/sabre/AltClick(mob/user)
+	if(contents.len)

```

Comment:
if(!alien() |& (incapacitated() || lying)
        Return //This isnt indented properly
Then rest of the altclick code afterwards in current setup.
at the start should catch some of the things BUT i highly suggest you test to see if morphs or anything like ghosts or simple mobs which you think shouldn't handle this are able to ineract after adding this check with a return to the start of the proc